LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
-Copyright (c) 2015 Moviepilot, Inc.
+Copyright (c) 2015-2017 Movie Pilot
```

Comment:
Although the company doesn't exist anymore, but I think we should keep the name as it was back then: `Moviepilot, Inc.`.